### PR TITLE
Update french translation + comments

### PR DESCRIPTION
**Pull request by mbousq**
Hi. This merge request updates the fr translation. I would also like to highlight :

#: src/models/persistent.js:37
msgid "Files"
#: src/models/persistent.js:40
msgid "e.g. .thunderbird"

--> Should this be Folder instead of Files ? The example mentions .Thunderbird

----

#: src/models/sessionBus.js:37 src/models/systemBus.js:37
msgid "Talks"
#: src/models/sessionBus.js:44 src/models/systemBus.js:44
msgid "Owns"

--> "Talks to" or perhaps "Uses" ? The system bus Talks To (or Uses) "org.freedesktop.secrets" and Owns whaterver.SearchProvider

----

Lastly, where can I translate titles like "Share", "Socket", "Allow" etc ?

Many thanks !

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -156,15 +157,16 @@ msgstr "e.g. .thunderbird"
 #: src/models/persistent.js:62
 msgid "List of homedir-relative paths created in the sandbox"
 msgstr ""
-"Liste des chemins relatifs au répertoir personnel créé dans le sac de sable"
+"Liste des chemins relatif créés dans la sandox, avec comme point de "
```
Comment (tchx84) [inline po/fr.po]: This probably won't fit in the section, and will get ellipsized?

File: po/fr.po [hunk 2]
```diff
@@ -306,44 +310,24 @@ msgstr "Dernière mise à jour"
...
 msgid "Show application in a software manager"
-msgstr "Montrer l'application dans un gestrionnaire de logiciels"
+msgstr "Afficher l’application dans un gestionnaire de logiciels"
 
 #: src/widgets/detailsButton.js:109
 msgid "No software manager found"
-msgstr "Aucun qestionnaire de logiciel trouvé"
-
-#: src/widgets/globalRow.ui:10 src/widgets/globalInfoViewer.ui:69
-msgid "All Applications"
```
Comment (tchx84) [inline po/fr.po:329]: ops, I think you're deleting these..

**Review by tchx84 — CHANGES_REQUESTED** [review]

**Comment by tchx84** [comment]
@mbousq it says 0 changes now :thinking: .. can you double check?

**Comment by mbousq** [comment]
Sorry it took me so long... Here are the corrections, ready to be merged. Thanks for reviewing.

File: po/fr.po (hunk 1 above)
Comment (mbousq) [reply po/fr.po]: Indeed, it gets ellipsized. I reverted that change

File: po/fr.po (hunk 2 above)
Comment (mbousq) [reply po/fr.po:329]: Fixed !

**Review by tchx84 — APPROVED** [review]

**Comment by tchx84** [comment]
Thank you!

**Pull request merged**